srcStatic/IniModuleLoader.cpp

```diff
@@ -18,9 +18,10 @@ void IniModuleLoader::LoadModules()
 void IniModuleLoader::LoadModule(std::string sectionName)
 {
 	IniModuleEntry moduleEntry;
+	moduleEntry.name = sectionName;
 
 	try {
-		LoadModuleDll(moduleEntry, sectionName);
+		LoadModuleDll(moduleEntry);
```

Comment:
Agreed that the code could be improved. I propose passing the module's handle out of LoadModuleDll as the solution. This way LoadModuleDll doesn't need the IniModuleEntry struct at all. Will push in a commit. A little different then proposed but I think results will be similar.

Reply:
Good point, that is another alternative. I didn't check closely, and had assumed both `handle` and `destroyModFunc` were set in that method, but they aren't. I suppose you could go either way. It may make sense to move setting the `destroyModFunc` field into that method.

The same could also be done for `initModFunc`. I considered storing that value too when I first wrote it. I didn't though, since there was no reason to store it after calling it. Might be cleaner code to store it in the struct though.